sunpy/net/dataretriever/tests/test_lyra_ud.py

```diff
@@ -36,12 +37,7 @@ def test_get_url_for_time_range(timerange, url_start, url_end):
     assert urls[-1] == url_end
 
 
-def test_get_url_for_date():
-    url = LCClient._get_url_for_date(parse_time((2013, 2, 13)))
-    assert url == 'http://proba2.oma.be/lyra/data/bsd/2013/02/13/lyra_20130213-000000_lev2_std.fits'
```

Comment:
Stupid question: Why was this test removed?

Reply:
The function `_get_url_for_date` no longer exists - it used to be a separate function that would return a url for a particular date and was called within `_get_url_for_timerange`  but now scraper takes care of it and its all done now inside `_get_url_for_timerange`